### DISCUSSION TITLE
Is it possible for users to choose between intents if the scores are similar, like in Power Virtual Agents?

### DISCUSSION THREAD

**Discussion by Leong-Qi-Jin** [microsoft/BotFramework-Composer#9235] [Q&A]
In Power Virtual Agents, if the bot is not sure about which 'Topic' aka 'Intent' is the right one, it gives the user a few options. How can I achieve that in Bot Framework Composer, or at least extend the bot with code?

**Comment by arunthachi** [comment]
we use LUIS in BFC and LUIS gives list of matching intents with potential confidence score. I assume you could use that.
